build_sysext

```diff
@@ -0,0 +1,103 @@
+
+Examples:
+
+build_sysext --board=amd64 python - builds python sysext for amd64 architecture.
+
+"
+
+show_help_if_requested "$@"
+
+# Parse command line
+FLAGS "$@" || exit 1
+
+eval set -- "${FLAGS_ARGV}"
+
+# N.B.  Ordering matters for some of the libraries below, because
+# some of the files contain initialization used by later files.
+. "${BUILD_LIBRARY_DIR}/toolchain_util.sh" || exit 1
+. "${BUILD_LIBRARY_DIR}/board_options.sh" || exit 1
+. "${BUILD_LIBRARY_DIR}/build_image_util.sh" || exit 1
+. "${BUILD_LIBRARY_DIR}/prod_image_util.sh" || exit 1
+. "${BUILD_LIBRARY_DIR}/dev_container_util.sh" || exit 1
+. "${BUILD_LIBRARY_DIR}/test_image_content.sh" || exit 1
+. "${BUILD_LIBRARY_DIR}/torcx_manifest.sh" || exit 1
+. "${BUILD_LIBRARY_DIR}/vm_image_util.sh" || exit 1
+
+BUILD_DIR="${DEFAULT_BUILD_ROOT}/images/${BOARD}/${IMAGE_SUBDIR}"
+
+
+cleanup() {
+    # Unmount the directory here
+    umount "${BUILD_DIR}/fs-root" "${BUILD_DIR}/install-root" "${BUILD_DIR}/workdir" || true
+}
+
+# Set up trap to execute cleanup() on script exit
+trap cleanup EXIT
+
+
+# export SYSEXTNAME=$2
+
+ARCH=[[ "$BUILD_DIR" == *arm64-usr* ]] && ARCH=arm64 || [[ "$BUILD_DIR" == *amd64-usr* ]] && ARCH=x86-64
+
+source /usr/lib/os-release
+
+
+unmount "${BUILD_DIR}/fs-root" "${BUILD_DIR}/install-root" "${BUILD_DIR}/workdir" || true
+
+
+# # with my 1 line patch before the rms in build_library/prod_image_util.sh (mksquashfs "${root_fs_dir}" "${BUILD_DIR}/configroot" "${BUILD_DIR}"/flatcar_production_image_sysext.sqsh)
+
+
+sudo mkdir "${BUILD_DIR}/fs-root"
+sudo mount -rt squashfs -o loop,nodev,noexec "${BUILD_DIR}/flatcar_production_image_sysext.sqsh" "${BUILD_DIR}/fs-root"
+VERSION_BOARD=$(awk -F= '$1=="FLATCAR_VERSION" { print $2 ;}' "${BUILD_DIR}/fs-root/usr/lib/os-release")
+if [ "$VERSION_BOARD" != "$VERSION" ]; then
+  echo "Version mismatch between board flatcar release and SDK container flatcar release"
+else
+  sudo mkdir "${BUILD_DIR}/install-root"
+  sudo mkdir "${BUILD_DIR}/workdir"
+  sudo mount -t overlay overlay -o lowerdir="${BUILD_DIR}/fs-root",upperdir="${BUILD_DIR}/install-root",workdir="${BUILD_DIR}/workdir" "${BUILD_DIR}/install-root"
+  sudo emerge-amd64-usr --getbinpkg "${SYSEXTNAME}"
+  sudo -E ROOT="/home/sdk/trunk/src/scripts/${BUILD_DIR}/install-root" PORTAGE_CONFIGROOT="/build/${BOARD}" FEATURES="-ebuild-locks" emerge --root-deps=rdeps --usepkgonly --verbose "${SYSEXTNAME}"
+  sudo umount "${BUILD_DIR}/install-root"
+  sudo umount "${BUILD_DIR}/fs-root"
+  sudo rm -rf install-root/{dev, etc, var}
+  sudo mkdir -p install-root/usr/lib/extension-release.d
+  sudo printf "ID=flatcar-releaseinfo\nVERSION_ID=${VERSION_ID}\nARCHITECTURE=${ARCH}" > "${BUILD_DIR}/install-root/usr/lib/extension-release.d/extension-release.${SYSEXTNAME}"
+  sudo mksquashfs install-root "${BUILD_DIR}/${SYSEXTNAME}.raw" --all-root
+
+  sudo rm -rf "${BUILD_DIR}/{fs-root, install-root, workdir}"
+
+  upload_image "${BUILD_DIR}/${SYSEXTNAME}.raw"
```

Comment:
If this is also for users, we could skip the upload step and let the build scripts handle it for the release case only?